functions/Get-DbaUserPermission.ps1

```diff
@@ -215,7 +215,8 @@ function Get-DbaUserPermission {
                     Stop-Function -Message "The database $db is not accessible" -Continue
                 }
 
-                $sql = [System.IO.File]::ReadAllText("$script:PSModuleRoot\bin\stig.sql")
+                $sqlFile = Join-Path -Path $script:PSModuleRoot -ChildPath "bin" -AdditionalChildPath "stig.sql"
```

Comment:
Ah, yeah approved to quickly.. should be changed to:
```suggestion
                $sqlFile = Join-Path -Path $script:PSModuleRoot -ChildPath "bin\stig.sql"
```

Reply:
the PR was created beecause \ will not work with Linux but it can be potentially `Resolve-Path (Join-Path -Path $script:PSModuleRoot -ChildPath "bin\stig.sql")`

``

Reply:
Yes, `Join-Path` handles converting the slash based on the OS for us.

Reply:
Yesss! Just tried it and did not know that. I thought it was just Resolve-path.